app/models/tslr_claim.rb

```diff
@@ -37,18 +37,36 @@ class TslrClaim < ApplicationRecord
   validates :claim_school,              on: :"claim-school", presence: {message: "Select a school from the list"}
   validates :qts_award_year,            on: :"qts-year", inclusion: {in: VALID_QTS_YEARS, message: "Select the academic year you were awarded qualified teacher status"}
   validates :employment_status,         on: :"still-teaching", presence: {message: "Choose the option that describes your current employment status"}
-  validates :full_name,                 on: :"full-name", presence: {message: "Enter your full name"}
-  validates :address_line_1,            on: :address, presence: {message: "Enter your building and street address"}
-  validates :address_line_3,            on: :address, presence: {message: "Enter your town or city"}
-  validates :postcode,                  on: :address, presence: {message: "Enter your postcode"}
+
+  validates :full_name,                 on: :"full-name",
+                                        presence: {message: "Enter your full name"},
+                                        length: {maximum: 200, message: "Full name must be 200 characters or less"}
+
+  validates :address_line_1,            on: :address, presence: {message: "Enter your building and street address"},
```

Comment:
We should probably be consistent with the line breaks here so it's easier to scan. On the validation rule above you have `presence` on a separate line

Reply:
I've had another stab at this, beause a lot have two validation contexts now, the standard 'save' and the custom one. I think it reads better now?